src/Response/Utils.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Pyrite\Response;
+
+use Pyrite\OutputBuilder\BinaryOutputBuilder;
+use Pyrite\OutputBuilder\StreamedOutputBuilder;
+
+class Utils
```

Comment:
It's recurrent naming in some OSS project for class who make easier our life but can be omitted to do it by the hand